ports/atmel-samd/boards/pyportal/pins.c

```diff
@@ -9,6 +9,7 @@
 // microcontroller.pin.
 STATIC const mp_rom_map_elem_t board_global_dict_table[] = {
     { MP_OBJ_NEW_QSTR(MP_QSTR_AUDIO_OUT), MP_ROM_PTR(&pin_PA02) },
+    { MP_OBJ_NEW_QSTR(MP_QSTR_SPEAKER), MP_ROM_PTR(&pin_PA02) },
```

Comment:
How about listing SPEAKER first so it's used when the Pin is printed out?

Reply:
Good question. Which is better to see in message? Actual schematic label? Or nominal common name? Yah, I dunno. Can see reasons for both. Let me know if you want it to be SPEAKER and I'll push that up. Easy peasy change.

Reply:
I prefer the most human readable form when the output isn't labeled on the board itself. SPEAKER is the preference, AUDIO_OUT is only for backwards compatibility.

Reply:
works for me. done.